test/math.jl

```diff
@@ -482,17 +494,26 @@ end
             @test_throws DomainError cospi(convert(T,Inf))
         end
         @testset "Check exact values" begin
+            # If the machine supports fused multiply add (fma), we require exact equality.
+            # Otherwise, we only require approximate equality.
+            if has_fma[T]
+                @info "On this machine, FMA is supported for $(T), so we will test for exact equality"
```

Comment:
These logs seem noisy. Perhaps change the testset name appropriately?

Reply:
We can just change it to `@debug` before merging. I just wanted to double-check that everything looked as expected

Reply:
IMO changing the testset name is better than `@debug` because if there's a failure it will show. `@debug` requires re-running with an env var

Reply:
Instead of putting it in the testset name, I've used the "testset context" feature (from https://github.com/JuliaLang/julia/pull/46138) to put it in the testset context. If there is a test failure, Julia will print the value of the `context` variable.

Reply:
The testset name isn't very accurate when `my_eq` isn't `==`.